File: Objc/Chapter 05/5.1/Classes/ViewController.m
//
//  ViewController.m
//
//  Created by Nick Lockwood on 03/02/2013.
//  Copyright (c) 2013 Charcoal Design. All rights reserved.
//

#import "ViewController.h"
#import <QuartzCore/QuartzCore.h>

@interface ViewController ()

@property (nonatomic, weak) IBOutlet UIView *layerView;

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	
    //rotate the layer 45 degrees
    CGAffineTransform transform = CGAffineTransformMakeRotation(M_PI_4);
    self.layerView.layer.affineTransform = transform;
    
    UIView* view1 = [UIView new];
    [self.view addSubview:view1];
    view1.frame = CGRectMake(100, 200, 100, 100);
    view1.backgroundColor = [UIColor redColor];
    
    view1.transform = transform;
    
}

@end

File: Swift/Chapter 03/3.3/OverlapTest/OverlapTest/ViewController.swift
//
//  ViewController.swift
//  OverlapTest
//
//  Created by pxl on 2017/7/10.
//  Copyright © 2017年 pxl. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    
    @IBOutlet weak var redView: UIView!
    
    @IBOutlet weak var greenView: UIView!

    
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
 
        self.greenView.layer.zPosition = 1.0
        
    }



}
